icubam/db/synchronizer.py

```diff
@@ -31,6 +31,7 @@
   'n_covid_deaths', 'n_covid_healed', 'n_covid_refused', 'n_covid_transfered',
   'create_date'
 ]
+BC_DATES_COLUMNS = ['create_date']
```

Comment:
Maybe put this constant inside the CSV class?  Not sure about my suggestion...

Reply:
This has to stay in sync with columns (if we change names, or add new date columns), so I think it is safer here.